### PR TITLE
Add pub_hwm and zmq_backlog to the tuning guide

### PR DESCRIPTION
**Pull request by moio**
# Description

Add pub_hwm and zmq_backlog knobs to the tuning guide.

# Target branches

Which product version does this PR apply to?

- [ ] Manager-4.0
- [x] Manager-4.1
- [x] Manager-4.2
- [x] Uyuni

# Links

part of https://github.com/SUSE/spacewalk/issues/11954

### CODE REVIEW

**Comment by moio** [comment]
FYI: @rjmateus @meaksh @vzhestkov

File: modules/large-deployments/pages/tuning.adoc [hunk 1]
```diff
@@ -502,6 +502,38 @@ If you see lines containing ``LOG:  temporary file:`` try raising this parameter
 
 
 
+[[pub-hwm]]
+=== `pub_hwm`
+
+[cols="1,1"]
+|===
+| Description          | The maximum number of outstanding messages sent by `salt-master`. If more than this number of messages need to be sent concurrently, communication with clients will slow down, potentially resulting in timeout errors during load peaks.
```
Comment (Loquacity) [inline modules/large-deployments/pages/tuning.adoc:510]:
```suggestion
| Description          | The maximum number of outstanding messages sent by `salt-master`. If more than this number of messages need to be sent concurrently, communication with clients slows down, potentially resulting in timeout errors during load peaks.
```

File: modules/large-deployments/pages/tuning.adoc [hunk 2]
```diff
@@ -502,6 +502,38 @@ If you see lines containing ``LOG:  temporary file:`` try raising this parameter
...
+|===
+
+
+
+[[zmq-backlog]]
+=== `zmq_backlog`
+
+[cols="1,1"]
+|===
+| Description          | The maximum number of allowed client connections that have started but not concluded the opening process. If more than this number of clients connects in a very short time frame, connections will be dropped and clients will experience a delay re-connecting.
```
Comment (Loquacity) [inline modules/large-deployments/pages/tuning.adoc:526]:
```suggestion
| Description          | The maximum number of allowed client connections that have started but not concluded the opening process. If more than this number of clients connects in a very short time frame, connections are dropped and clients experience a delay re-connecting.
```

**Review by Loquacity — CHANGES_REQUESTED** [review]
minor edits for future tense

File: modules/large-deployments/pages/tuning.adoc [hunk 3]
```diff
@@ -502,6 +502,38 @@ If you see lines containing ``LOG:  temporary file:`` try raising this parameter
 
 
 
+[[pub-hwm]]
+=== `pub_hwm`
+
+[cols="1,1"]
+|===
+| Description          | The maximum number of outstanding messages sent by `salt-master`. If more than this number of messages need to be sent concurrently, communication with clients will slow down, potentially resulting in timeout errors during load peaks.
+| Tune when            | xref:client-count[Client count] increases significantly and `Salt request timed out. The master is not responding` errors appear when pinging minions during a load peak.
```
Comment (keichwa) [inline modules/large-deployments/pages/tuning.adoc:511]:
```suggestion
| Tune when            | xref:client-count[Client count] increases significantly and ``Salt request timed out. The master is not responding.`` errors appear when pinging minions during a load peak.
```
I think single tics don't work with adoc.  If yes, we must adjust it later on, too.
We usually call minions "clients".  I'm not sure whether we want to do the same here.

**Review by keichwa — APPROVED** [review]

**Review by Loquacity — APPROVED** [review]

**Pull request merged**